colossalai/inference/engine/policies/llama.py

```diff
@@ -20,20 +20,17 @@
 from ..modeling.llama import LlamaInferenceForwards
 
 try:
-    from colossalai.kernel.triton import rmsnorm_forward
-
+    from lightllm.models.llama.triton_kernel.rmsnorm import rmsnorm_forward as lightllm_rmsnorm_forward
     HAS_TRITON_RMSNORM = True
 except:
-    print("you should install triton from https://github.com/openai/triton")
+    print("you should install triton from GitHub: https://github.com/ModelTC/lightllm.git@ece7b43f8a6dfa74027adc77c2c176cff28c76c8")
```

Comment:
This is still weird, the message it to install triton, but the github is lightllm. Meanwhile, can you please provide an installation command directly for better user experience?

Reply:
What if a user has installed lightllm but not triton, what will happen?

Reply:
fixed some comments. added pip install command

Reply:
installing lightllm will automatically install triton